code/java/FlintParser/src/test/kotlin/org/discpl/flint/FlintParserTest.kt

```diff
@@ -192,4 +193,36 @@ internal class FlintParserTest {
             }
         }
     }
+
+    @Test
+    fun getLanguage(){
+        this::class.java.classLoader.getResourceAsStream("projection-and-create.flint.json").use {
+            InputStreamReader(it).use {
+                val text = it.readText()
+                val flintParser = FlintParser(text)
+                val language = flintParser.getLanguage()
+                val correctLanguage: Language = Language("Dutch","naam","daden","feiten",
+                    "plichten","functie","referentie","taak-eigenaar","eiser",
+                    "maak","beindig","taak-onderdelen","uitleg","actie-ondernemer",
+                    "actie","object","ontvanger","pre-conditie","taal","bronnen",
+                    "tekst","Vind lijn in Bron","geldig vanaf","geldig tot")
+                println("[LANGUAGE_CHECK] $language")
+                assertEquals(correctLanguage,language)
+            }
+        }
+        this::class.java.classLoader.getResourceAsStream("test-Vreemdelingenwet.flint.json").use {
+            InputStreamReader(it).use {
+                val text = it.readText()
+                val flintParser = FlintParser(text)
+                val language = flintParser.getLanguage()
+                val correctLanguage: Language = Language("English","name","acts","facts",
```

Comment:
No need to change this but expected is the better term to use instead of correct